recipes/hepdata-validator/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "hepdata-validator" %}
+{% set version = "0.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hepdata_validator-{{ version }}.tar.gz
+  sha256: d93a23e4eeb41df03011494482b619ac59088f13054bd389718111d135f2ab10
+
+build:
+  entry_points:
+    - hepdata-validate=hepdata_validator.cli:validate
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pytest
+    - pip
+  run:
+    - python >=3.6
+    - click
+    - jsonschema
+    - packaging
+    - pyyaml >=5.4.1
+    - requests
+
+test:
+  imports:
+    - hepdata_validator
+  commands:
+    - pip check
+    - hepdata-validate --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/hepdata/hepdata-validator
+  summary: JSON schema and validation code for HEPData submissions
+  license: GPL-2.0
```

```suggestion
  license: GPL-2.0-or-later
```

@GraemeWatt can you also comment if you would prefer https://spdx.org/licenses/GPL-2.0-only.html or https://spdx.org/licenses/GPL-2.0-or-later.html?

---

I'm not sure I understand the advantage of `GPL-2.0-or-later`.  The [license](https://github.com/HEPData/hepdata-validator/blob/main/LICENSE.txt) distributed with the original repository since its creation in 2015 is `GPL-2.0`.  But if you think `GPL-2.0-or-later` is better, then I'll go with your decision.

---

Thanks. I went with `GPL-2.0-only`. :+1: